<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acoustic Emanations Tool</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxTrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Record</string>
    </property>
    <widget class="QLineEdit" name="lineEditKeyVal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRecNb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRec">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBarRecord">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelKeyName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Key name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelIndex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Index:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>881</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>Model</string>
    </property>
    <widget class="MplWidget" name="MplWidgetMatrix" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>871</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonModel">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>570</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Model</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTrainAccuracy">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>580</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Train accuracy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTestAccuracy">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>600</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Test accuracy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPreTrainAccuracy">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>560</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre-train accuracy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFeatures">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Features:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBarCompute">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>600</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidgetSpectrum" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadWaveFile"/>
    <addaction name="actionFolderSelect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFolderSelect">
   <property name="text">
    <string>Folder Select</string>
   </property>
  </action>
  <action name="actionLoadWaveFile">
   <property name="text">
    <string>Load wave file</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
